core/services/vrf/listener_v1.go

```diff
@@ -375,7 +382,15 @@ func (lsn *listenerV1) ProcessRequest(req *solidity_vrf_coordinator_interface.VR
 		}
 		return nil
 	}); err != nil {
-		lsn.l.Errorw("Failed executing run", "err", err)
+		lsn.l.Errorw("Failed executing run",
+			"err", err,
+			"reqID", hex.EncodeToString(req.RequestID[:]),
+			"reqTxHash", req.Raw.TxHash)
+	} else {
+		lsn.l.Infow("Executed fulfillment run",
```

Comment:
This is going to be way too noisy, can we downgrade to Debug please

Reply:
I think will have to disagree, we have this for V2 and it's definitely useful.

Reply:
Done